app/blog-content.css

```
@@ -34,9 +34,38 @@
   /* Table styling for markdown tables */
   .blog-content table {
     @apply border-collapse border border-secondary-200 dark:border-secondary-700 my-6;
-    width: auto;
-    min-width: 300px;
+    display: block;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)